src/components/LifeEventsListView.vue

```diff
@@ -118,6 +158,26 @@ export default {
             if (this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
             if (! this.includeUpcomingEvents && !isFuture(e.redZone.start)) return true
           })
+          .filter(e => {
+            if(this.filterTag === undefined) return true
+            console.log(this.filterTag)
+            if(e.title.includes("#"+this.filterTag)) return true
+            return false
+          })
+    },
+    listOfCurrentTags() {
```

Comment:
Hier noch eine Alternative. Ist noch mehr "fancy". Nur mal um nochmal was Neues zu sehen. Im Prinzip löst es die doppelten `forEach` und macht es IMHO noch etwas leserlicher:

```
listOfCurrentTags() {
      return new Set( // a set holds only unique values
          this.sortedEvents(this.sortBy)
            // we want to transform an input collection into a output collection
            // this transformation can be done by `map()`
            .map(event => event.title) // we just need the `title` for future computation
            .filter(title => title.includes('#')) // skip all titles w/o the `#` char
            .flatMap(title => {
              let splitted = title.split('#')
              // with `splice` we can take the last x elements (aka everything after the first element)
              return splitted.splice(1, splitted.length - 1)
              // this returns an array. this would result in an array of arrays `[[a,b],[a],[c,e],...]`
              // but we wanna hava a "flat" view. therefore we use `flatMap()` which will result in
              // `[a,b,a,c,e]`
            })
      )
    }
```

Reply:
Nice, danke für das Review. Ich werde dran arbeiten, wenn ich Zeit finde :-D